pkg/services/rpcsrv/error.go

```diff
@@ -44,8 +44,6 @@ func getHTTPCodeForError(respErr *neorpc.Error) int {
 	switch respErr.Code {
 	case neorpc.BadRequestCode:
 		httpCode = http.StatusBadRequest
-	case neorpc.InvalidRequestCode, neorpc.RPCErrorCode, neorpc.InvalidParamsCode:
-		httpCode = http.StatusUnprocessableEntity
```

Comment:
fixed

Reply:
Please, read the https://github.com/nspcc-dev/.github/blob/master/git.md#commits. Commit header prefix should be a package (or a large part of a project) where changes are made, in your case it's `rpcsrv`. Infinitive form in the commit header is welcomed, in your case it's `refactor`. The commit header may be more specific, and the commit bodu may contain a couple of words about this change. It can be something like this:
```
rpcsrv: remove default request HTTP codes from switch statement

No functional changes, just a refactoring.
```

Reply:
Thank you for the link and explanation